content/_data/sidebar.json

```diff
@@ -372,7 +372,8 @@
           "children": [
             {
               "title": "Table of Contents",
-              "redirect": "/api/table-of-contents"
+              "redirect": "/api/table-of-contents",
+              "slug": "table-of-contents"
```

Comment:
I don't think it's valid for a sidebar item to have both a `slug` and a `redirect`. [This code](https://github.com/cypress-io/cypress-documentation/blob/master/components/sidebar/CollapsibleSidebarSection.vue) always shows them as mutually exclusive. I think it would be confusing and/or problematic for a given node to have both properties.

Reply:
@cowboy thanks for pointing this out. I will take a look and make changes as needed.

Reply:
@cowboy I have reverted the sidebar.json and updated the script to handle these 2 children accordingy.